litellm/integrations/traceloop.py

```diff
@@ -1,10 +1,26 @@
+import traceback
+
+
 class TraceloopLogger:
     def __init__(self):
-        from traceloop.sdk.tracing.tracing import TracerWrapper
-        from traceloop.sdk import Traceloop
+        try:
+            from traceloop.sdk.tracing.tracing import TracerWrapper
+            from traceloop.sdk import Traceloop
+
+            Traceloop.init(app_name="Litellm-Server", disable_batch=True)
+            self.tracer_wrapper = TracerWrapper()
+        except Exception as e:
```

Comment:
We should avoid catching the generic exception instead if we wanted to warn and suggest package is missing we should use ModuleNotFoundError exception.
```
ModuleNotFoundError: No module named 'traceloop'
```